### PR TITLE
DEVC: Fix check if package is empty

### PR DESCRIPTION
**Pull request by mbtools**
SOTR object is linked to SAP package and removed together with the package on uninstall. Therefore, it should not be taken into account when checking if a package is empty.

### CODE REVIEW

**Comment by larshp** [comment]
as far as I understand SOTR is a generic thing that is used for multiple objects? 

This change works only under the assumption that all SOTR objects are tied to the DEVC?

**Comment by mbtools** [comment]
Exactly. If any other object - like a class - exists, `rv_is_empty` will be false. If there's only `DEVC` - with or without `SOTR` - left, then `rv_is_empty` should be true meaning the package can be deleted.

**Comment by larshp** [comment]
but then the package contains a SOTR object, and I would expect it not to be possible to delete the package?

**Comment by larshp** [comment]
oh, I think I read the code wrong, it doesnt skip any SOTR object

but only if the SOTR object belongs/is part of the DEVC

**Comment by larshp** [comment]
I think the comment `SOTR is linked to SAP packages (DEVC)` can be misleading, SOTR is not linked to only DEVC on a general note

But if the SOTR is linked to the current DEVC, then it can be skipped

File: src/objects/zcl_abapgit_object_devc.clas.abap [hunk 1]
```diff
@@ -91,11 +91,12 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
       RETURN.
     ENDIF.
 
+    " SOTR is linked to SAP packages (DEVC)
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_devc.clas.abap:94]:
```suggestion
    " Ignore the SOTR if is linked to the current SAP package (DEVC)
```

**Review by larshp — CHANGES_REQUESTED** [review]

**Review by larshp — APPROVED** [review]

**Pull request merged**